src/test/java/com/google/javascript/gents/multiTests/provide_imports/default_exports1_keep.js

```diff
@@ -0,0 +1,2 @@
+goog.provide('d.export1');
```

Comment:
provide_imports test is already very messy and hard to understand what is being tested. 

Can you either:
- extend jslib_imports, I think it's lib.B and lib.C are basically your export1 and export2.
- create a new test.

Reply:
Moved to jslib_imports.